### ISSUE TITLE
[dev_react] Critical bug in use of Redux toolkit

### ISSUE THREAD

**Issue by aiyan** [Path-Check/safeplaces-frontend#71]
## Expected behavior
The site should load properly.
## Current behavior
An error is thrown, interrupting the app's execution.

![error](https://user-images.githubusercontent.com/48393820/81511804-e083f100-92e9-11ea-9fc8-7f68cb0fbec8.PNG)
## Steps to reproduce
Checkout 8b7e125, the latest commit in `dev_react`, and run `yarn start`.
## Possible explanation
The previous commit, b90b411, runs properly. @Utzel-Butzel, can you reproduce this error?
## Environment
|Context|Details|
|-|-|
Google Chrome | 81.0.4044.138 (Official Build) (64-bit) (cohort: Stable)
OS | Windows 10 OS Version 1909 (Build 18363.778)
JavaScript | V8 8.1.307.32

**Comment by mundanelunacy** [comment]
@aiyan 
https://github.com/Path-Check/safeplaces-frontend/commit/8b7e1259f1700c448cb7de40d1cda22ca9de8e78?diff=unified  
works for me on my mac with no issues.  

Just a guess but the error you posted looks like it could be an issue with your Babel setup.  It it set up globally on your environment?

**Comment by aiyan** [comment]
@mundanelunacy I had installed all of the dependencies with `yarn install`. Just to test, I installed `@babel/core` as a development dependency, and `create-react-app` warns me to use their version of babel instead of manually installing it.

@Utzel-Butzel I forwarded my head to your latest commit on `dev_react` and I still receive the same error. Is there a reason why `patients` might be undefined or null?
